.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -3264,6 +3264,7 @@ stages:
               -e IncludeTestsRequiringDocker=true \
               -e Filter=$(IntegrationTestFilter) \
               -e SampleName=$(IntegrationTestSampleName) \
+              -e USE_FULL_TEST_CONFIG=$(isMainOrReleaseBranch) \
```

Comment:
Was this one intentionally set, or missed in the conversion to `RequiresThoroughTesting()`?

Reply:
Hmm I must've missed it somehow, thanks for catching!

Reply:
```suggestion
```